controllers/sendMail.js

```diff
@@ -0,0 +1,33 @@
+const nodemailer = require('nodemailer');
+require('env2')('./config.env');
+
+module.exports = (req, res) => {
+  const { email, message, name } = req.body;
+  const transporter = nodemailer.createTransport({
+    service: 'gmail',
+    secure: false,
+    port: 25,
+    auth: {
+      user: 'noreply.hbnb@gmail.com',
```

Comment:
This should be an environment variable along with the password

Reply:
Done.